models/article.models.js

```diff
@@ -43,3 +43,31 @@ exports.updateArticleById = async (id, body) => {
     return updatedObj[0];
   }
 };
+
+exports.fetchArticles = async () => {
+  const { rows: articles } = await db.query(
+    "SELECT * FROM articles ORDER BY created_at DESC;"
+  );
+
+  if (articles.length === 0) {
+    return Promise.reject({
+      status: 404,
+      msg: "No article found with that id!",
+    });
+  }
+  const { rows: comments } = await db.query(
+    "SELECT * FROM articles INNER JOIN comments ON articles.article_id = comments.article_id"
+  );
```

Comment:
this works fine, but is there a way to get this information in one request to the db rather than two?